test/integration/rebase_patches

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# rebase a set of patches, assumes the kernel has already been downloaded into
+# the kpatch $CACHEDIR.Output patches go into ./${ID}_${VERSION_ID}/
+#
+# Example:
+#
+# ./rebase_patches old_dir/*.patch
```

Comment:
I like this.  Can you rename it to `rebase-patches` to be consistent with other kpatch program names?